src/System.Numerics.Vectors/tests/Vector3Tests.cs

```diff
@@ -61,7 +61,6 @@ public void Vector3GetHashCodeTest()
             Assert.False(v4.GetHashCode() == v8.GetHashCode());
             Assert.False(v7.GetHashCode() == v6.GetHashCode());
             Assert.False(v8.GetHashCode() == v6.GetHashCode());
-            Assert.True(v8.GetHashCode() == v7.GetHashCode());
```

Comment:
Vectors v8 and v7 are different, so hash function could be the same or not. Why checks like this are needed? I think that they don't mean anything.